fiatconnect-api.md

```diff
@@ -2110,7 +2110,8 @@ An enum listing the frequency, or how often, a particular fee needs to be paid.
 	`MobileMoney`,
 	`DuniaWallet`,
 	`IBANNumber`,
-	`IFSCAccount`
+	`IFSCAccount`,
+	`USDAccount`
```

Comment:
can you update the swagger spec with this too?

Reply:
done.